firedrake_configuration/__init__.py

```diff
@@ -9,25 +9,14 @@
 
 # Attempt to read configuration from file.
 try:
-    with open(os.path.join(os.path.dirname(__file__),
-                           "configuration.json"), "r") as f:
-        _config = json.load(f)
+        with open(os.path.join(os.environ["VIRTUAL_ENV"],
```

Comment:
I had problems with that path name. It gives something like 
/Users/sv2518/firedrake_testbuildownpetsc/firedrake .configuration.json.

Reply:
Ah, of course. The git pull changes `firedrake_configuration` before the configuration file gets moved. Thanks for catching this @sv2518. I think I know how to fix it.